tests/resources/serializers/test_datacite_serializer.py

```diff
@@ -207,7 +207,7 @@ def test_datacite43_serializer(running_app, full_record):
             },
             {
                 "identifier": "1924MNRAS..84..308E",
-                "identifierType": "bibcode"
+                "identifierType": "Bibcode"
```

Comment:
There are many cases where this labelling is being tested (e.g. `doi` to `DOI`) but this is a clear one.